### PR TITLE
v2.0.0

### PR DESCRIPTION
**Pull request by hwangm**
- Update default API version to 2018-02-08
- Update documentation to reflect changes to config location
- Update examples to use `estimated_days` instead of `days`

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,6 +1,6 @@
 The MIT License
 
-Copyright (c) 2014 Shippo (http://goshippo.com)
+Copyright (c) 2020 Shippo (http://goshippo.com)
```
Comment (assislucas) [inline LICENSE]: I don't know how licenses work but are we supposed to update every year?

**Comment by assislucas** [comment]
Also, do we need to do this one? `Update docs and add "Migrating to v2" section`

**Comment by assislucas** [comment]
If the changes are small enough maybe not, up to you.

File: .travis.yml [hunk 2]
```diff
@@ -1,11 +1,8 @@
 language: python
```
[the remark this answers is not in the document]
Comment (hwangm) [reply .travis.yml]: Travis is still working: https://travis-ci.org/goshippo/shippo-python-client/builds/634977787?utm_source=github_status&utm_medium=notification

File: LICENSE (hunk 1 above)
Comment (hwangm) [reply LICENSE]: I believe we should be updating the copyright every year but a quick google search yields conflicting viewpoints - some say copyright should show the year of origination, some say copyright should be the most recent update date

**Comment by hwangm** [comment]
> Also, do we need to do this one? `Update docs and add "Migrating to v2" section`

I think malcolm already completed this in rc1

**Pull request merged**